examples/virtual_keyboard/rust/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+# Copyright © SixtyFPS GmbH <info@slint-ui.com>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+[package]
+name = "virtual_keyboard"
+version = "1.0.0"
+authors = ["Slint Developers <info@slint-ui.com>"]
+edition = "2021"
+build = "build.rs"
+publish = false
+license = "GPL-3.0-only OR LicenseRef-Slint-commercial"
+
+[[bin]]
+path = "main.rs"
+name = "virtual_keyboard"
+
+[dependencies]
+slint = { path = "../../../api/rs/slint" }
+
+[build-dependencies]
+slint-build = { path = "../../../api/rs/build" }
+
+# Remove the `#wasm#` to uncomment the wasm build.
```

Comment:
It looks like the index.html is missing.
Or perhaps we shouldn't support wasm for this example?

Reply:
Good question. As example I think it's good but it could be bring up problems with mobile keyboards. Maybe we should skip this for now (mean remove wasm form the example).